src/main/window-lifecycle.js

```diff
@@ -94,5 +94,10 @@ export async function createMainWindow({ dimensions, positions }: any, settings:
     mainWindow = null;
   });
 
+  mainWindow.webContents.on('new-window', (event, url) => {
+    event.preventDefault();
+    shell.openExternal(url)
```

Comment:
⚠️  make sure to check the url starts with `https://` or `http://` but nothing else.

Reply:
roger that

Reply:
all good :)